source/voice_control/s3-box-customize.markdown

```diff
@@ -180,20 +180,25 @@ If your images have transparency, you can define the background color in the con
 
 ## Customizing on-device wake words (microWakeWord)
 
-If you are running the latest version of ESPHome on your device, you can already process your wake word on your S3-BOX (instead of running it on the Home Assistant server). This is done using the [microWakeWord](https://github.com/kahrendt/microWakeWord) model. By default, the wake word then is *Okay Nabu*. If you want to change the on-device wake word to *Hey Jarvis* or *Alexa*, follow these steps.
+You can change the on-device wake word (microWakeWord) that is used on your S3-BOX-3.
+
+Do the following steps if one of the following options apply:
+
+- **Option 1**: Your first installation of the S3-BOX-3 is ESPHome 2024.2. In this case, it already supports the on-device wake word *Hey Jarvis*. Follow these steps only if you want to change the wake word. The currently supported on-device wake words are *Okay Nabu*, *Hey Jarvis*, or *Alexa*.
+- **Option 2**: You have an existing S3-BOX-3 running an older version of ESPHome. In this case, the S3-BOX-3 is not updated automatically to support on-device wake word. Follow these steps if you want update the firmware to process your wake word on your S3-BOX-3, instead of running it on the Home Assistant server. This is done using the [microWakeWord](https://github.com/kahrendt/microWakeWord) model.
```

Comment:
```suggestion
You can change the on-device wake word (microWakeWord) that is used on your S3-BOX-3.

Do the following steps if one of the following options apply:

- **Option 1**: Your first installation of the S3-BOX-3 is ESPHome 2024.2. In this case, it already supports the on-device wake word *Hey Jarvis*. Follow these steps only if you want to change the wake word. The currently supported on-device wake words are *Okay Nabu*, *Hey Jarvis*, or *Alexa*.
- **Option 2**: You have an existing S3-BOX-3 running an older version of ESPHome. In this case, the S3-BOX-3 is not updated automatically to support on-device wake word. Follow these steps if you want to update the firmware to process your wake word on your S3-BOX-3, instead of running it on the Home Assistant server. This is done using the [microWakeWord](https://github.com/kahrendt/microWakeWord) model.
```